### DISCUSSION TITLE
怎么定义烂尾？

### DISCUSSION THREAD

**Discussion by gohna** [WeNeedHome/SummaryOfLoanSuspension#530] [Q&A]
看了几个有图片的停贷帖，有一些是2022、2023甚至2024才交付的，这种也算烂尾吗，而且干工程的应该都知道，一般都会拖一段时间才交付吧，这种现象貌似也持续很久了，有的时候盖得快交的早不一定盖得好，因为从工程管理上来说这样成本高，那为了控制成本可想而知...?
合理讨论，我是觉得现在工程很多施工单位自己也不好过，如果矛头不对准真正的烂尾、恰烂钱的人，可能会导致混淆视听，真正的罪犯资本转移，而那些只是因为回款或者别的原因短时间、晚交付的工程队会受到重创，或者他们扛不住，偷工减料再贿赂质检，造成更大的损失...?

**Comment by wedxz** [comment]
是谁的问题谁解决,总不能一直让最弱势的老百姓当冤大头吧

**Reply by gohna** [reply]
我没有说老百姓要当冤大头，我的意思是看到有一些2023、2024才交付的也来凑热闹，觉得这样不合理，量大但上面一看这不都还没到时期，然后各种打击威胁，而真正烂尾的没人管，或者量太大，一件一件处理，就会很慢，反而给一些恰烂钱的人时间，让他们转移资产逃脱。我是这个意思

**Comment by b4o** [comment]
超过半年一年没有复工的，真等到交付的时候才去关注，那转出去的钱都已经花完了，已经发生了的风险那就是灾祸了

**Comment by C-Joey** [comment]
那得看哪个公司了 像恒大这种只要是没建好基本上就算是烂尾了  比如一个公司2021年的项目已经烂尾了 难道还指望其他项目2023年的能够完工？
